contracts/test/GovernorSequentialProposalIdUpgradeable.t.sol

```diff
@@ -30,7 +30,7 @@ contract GovernorSequentialProposalIdUpgradeableTest is CompoundGovernorTest {
 
 contract ProposalCount is GovernorSequentialProposalIdUpgradeableTest {
     function test_ReturnsCorrectProposalCount() public {
-        assertEq(governor.proposalCount(), compoundGovernorBravo.proposalCount());
+        assertEq(governor.proposalCount(), compoundGovernorBravo.proposalCount() + 1);
```

Comment:
Check my logic on this, but I think we have parity with GovernorBravo right now. Because of the indexing from 1, which we are basically replicating in our code. We currently return the "nextProposalID" as the return value for `proposalCount()`.

For instance, if the last proposal created had an ID of 301, and the first 200 belonged to GovernorBravo (1 to 200) and last 101 belonged to our new CompoundGovernor (201 to 301), our `proposalCount()` would return 301, which is correct, and the next proposal ID would be 302, and after another proposal is created, the `proposalCount()` function would return 302.

I think if we add 1 to what our `proposalCount()` function returns, our count will be 1 too high.